test/model-definition.js

```diff
@@ -13,6 +13,7 @@ var ModelRelation = app.models.ModelRelation;
 var TestDataBuilder = require('./helpers/test-data-builder');
 var ref = TestDataBuilder.ref;
 var ConfigFile = app.models.ConfigFile;
+var nodePath = require('path');
```

Comment:
This is confusing, (almost all) Node.js code uses `var path = require('path')`. Can you please rename the other variables that are clashing with the `path` name?